content/en/docs/concepts/security/overview.md

```diff
@@ -0,0 +1,151 @@
+vulnerable to a resource exhaustion attack, by not putting resource limits on
+Service B you run the risk of also compromising Service A. Below is a table of
+links of things to consider when securing workloads running in Kubernetes.
+
+Area of Concern for Workload Security | Recommendation |
+------------------------------ | ------------ |
+RBAC Authorization (Access to the Kubernetes API) | https://kubernetes.io/docs/reference/access-authn-authz/rbac/
+Authentication | https://kubernetes.io/docs/reference/access-authn-authz/controlling-access/
+Application secrets management (and encrypting them in etcd at rest) | https://kubernetes.io/docs/concepts/configuration/secret/ <br> https://kubernetes.io/docs/tasks/administer-cluster/encrypt-data/
+Pod Security Policies | https://kubernetes.io/docs/concepts/policy/pod-security-policy/
+Quality of Service (and Cluster resource management) | https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/
+Network Policies | https://kubernetes.io/docs/concepts/services-networking/network-policies/
+TLS For Kubernetes Ingress | https://kubernetes.io/docs/concepts/services-networking/ingress/#tls
+
+
+
+## Container
+
+The price of admission to a Kubernetes cluster is a container. Because of this,
+there are certain security considerations that must be taken into account in order
+to benefit from the workload security primitives of Kubernetes. Container security
+is also outside the scope of this article, but we will provide a table of general
+recommendations and links for further exploration of this topic.
+
+Area of Concern for Containers | Recommendation |
+------------------------------ | ------------ |
+Container Vulnerability Scanning and OS Dependency Security | As part of an image build step or on a regular basis you should scan your containers for known vulnerabilities with a tool such as [CoreOS's Clair](https://github.com/coreos/clair/)
+Image Signing and Enforcement | Two other CNCF Projects (TUF and Notary) are useful tools for signing container images and maintaining a system of trust for the content of your containers. If you use Docker, it is built in to the Docker Engine as [Docker Content Trust](https://docs.docker.com/engine/security/trust/content_trust/). On the enforcement piece, [IBM's Portieris](https://github.com/IBM/portieris) project is a tool that runs as a Kubernetes Dynamic Admission Controller to ensure that images are properly signed via Notary before being admitted to the Cluster.
+Disallow privileged users | When constructing containers, consult your documentation for how to create users inside of the containers that have the least level of operating system privilege necessary in order to carry out the goal of the container.
+
+## Code
+
+Finally moving down into the application code level, this is one of the primary attack
+surfaces over which you have the most control. This is also outside of the scope
```

Comment:
clean up: `but we recommend a few of the following [what]`?

Reply:
Cleaned up